tests/cypress/e2e/textMarks.cy.js

```diff
@@ -0,0 +1,15 @@
+describe('textMarks Snapshots', () => {
+    it('Screenshots are matching up', () => {
+      cy.visit('http://localhost:8888')
+        .then(() => {
+            cy.wait(10000); // TODO (Fede Jul 24 2023) Remove explicit waits when we found an onRender/onAnimateTransform, etc.
+  
+            cy.get(".center-track")
```

Comment:
Does using timeout inside the get work instead of waiting for a random time at the start?
If the reason for waiting is to wait till the ".center-track" is visible, It would be better to add a timeout in the get. WDYT?

```suggestion
            cy.wait(10000); // TODO (Fede Jul 24 2023) Remove explicit waits when we found an onRender/onAnimateTransform, etc.
  
            cy.get(".center-track", { timeout: 10000 })
```

Reply:
This did work in your end? :thinking: 

I guess that the DOM node is rendered but the svg is not drawn yet. So we need to catch with event that goslingjs is firing (if any) and work from there.

Reply:
No, I did not check it. I see. I don't know cypress that well to check for the SVG content but feel that it should be possible.

Reply:
@vedhav 
I misread the threshold in the cypress configuration. I assumed that 0.1 was 10 percent but it was actually 0.1 %
I set up the threshold to 5%. So you can check the tests back again